doc/nrf/device_guides/working_with_nrf/nrf91/nrf9161.rst

```diff
@@ -24,6 +24,8 @@ The board controller controls signal switches on the nRF9161 DK and can be used
 For a complete list of configuration options available, see the nRF9161 DK board control section in the nRF9161 DK User Guide.
 
 The nRF5340 IMCU comes preprogrammed with J-Link Segger OB and board controller firmware.
+If you want to change the default configuration of the board controller firmware, you can use the Board Configurator in `nRF Connect for Desktop`_ to modify the configuration of the board controller.
```

Comment:
```suggestion
If you want to change the default configuration of the DK, you can use the Board Configurator application in `nRF Connect for Desktop`_ .
```

Reply:
Updated. The nRF Connect for Desktop applications are named as apps in the NCS docs. So, updated accordingly.